DIP-16: Universally verifiable off-chain polls for Devcon 6 (#53)
efdevcon/DIPs: DIPs/DIP-16.md

Our proposal entails the development of a privacy-centric, scalable, affordable and universally verifiable governance layer complete with a great UX/UI. This will allow Devcon attendees to be heard and enable a wide range of possibilities for the event's organization while stimulating the crowd's collective knowledge.

We will achieve this objective by linking the efforts of two experienced teams in the field of governance and permissionless census creation: Vocdoni and POAP.

With this proposal, attendees -who are POAP tokens holders- will be able to vote in the many voting processes created by Devcon organizers using [Vocdoni Bridge](https://www.notion.so/Introducing-Vocdoni-Bridge-cf7e73d38c4a45788358e9a1497cdf19), a token-based voting system relying on the Vocdoni Open Stack.

For the Q1 2021 pilot project, we propose to use the dataset of Devcon Osaka attendees (which would likely provide an accurate representation of Devcon 6's attendees). Token-holders will be able to vote using some of the most common wallets including Metamask and Web3Modal. In the following months we'll focus on developing integrations with several others, such as Alphawallet.

> The voting tool will rely on the Vocdoni Open Stack, a universally verifiable, privacy-centric, and scalable voting system, as a Layer2 for Ethereum and xDai-chain laveraging on three main technlogies: Tendermint, libp2p and IPFS. With this stack we will be able to host a transparent governance process which includes complex voting mechanisms like quadatic or range voting. The deterministic results (computed by all layer2 blockchain nodes) can be send back to Ethereum in order to execute binding smart contract operations. In the near future, combined with [zk-SNARKs](https://docs.vocdoni.io/#/architecture/protocol/franchise-proof?id=zk-rollups-proposal-for-anti-coercion-scalable-and-deterministic-execution) anonymous voting will be possible as well.

Reviewer: ```suggestion
The voting tool will rely on the Vocdoni Open Stack, a universally verifiable, privacy-centric, and scalable voting system, as a Layer2 for Ethereum and xDai-chain laveraging on three main technologies: Tendermint, libp2p and IPFS. With this stack we will be able to host a transparent governance process which includes complex voting mechanisms like quadratic or range voting. The deterministic results (computed by all layer2 blockchain nodes) can be send back to Ethereum in order to execute binding smart contract operations. In the near future, combined with [zk-SNARKs](https://docs.vocdoni.io/#/architecture/protocol/franchise-proof?id=zk-rollups-proposal-for-anti-coercion-scalable-and-deterministic-execution) anonymous voting will be possible as well.
```

2. Attendees login with their wallet where they hold the POAP tokens.
3. Attendees vote on the speakers of each track using quadratic voting.
4. Attendees and third parties can validate the results by using a third party node or by running their own node

Note: The current Vocdoni Platform includes a client App available in Google and Apple stores. However DevCon users won't be using this App client but rather the new web based DApp being developed ([Vocdoni Bridge](https://www.notion.so/Introducing-Vocdoni-Bridge-cf7e73d38c4a45788358e9a1497cdf19)) in order to reduce friction and to make the platform easy to integrate with any Dapp wallet client (Alphawallet, Status, etc.).

Users and organizers will be able to track the votes using the [Vochain explorer](https://explorer.vocdoni.net), a tool to draw data from the voting processes on the blockchain and make it human-readable and searchable.

- Has any part of this proposal been implemented at other events? If so, please describe how it went.

POAP's Tokens technology has been used in many physical and virtual events, including DevCon, EthCC, EthDenver, EDcon and Dappcon. There are already more than 650 POAPs representing events.

> The Vocdoni Open Stack has been used successfuly by various organizations (Municipalities, associations, cooperatives, companies and informal organizations) to organize Signalling polls, Event polls, Annual General Meetings and elections.

Reviewer: ```suggestion
The Vocdoni Open Stack has been used successfully by various organizations (Municipalities, associations, cooperatives, companies and informal organizations) to organize Signalling polls, Event polls, Annual General Meetings and elections.
```

The largest cultural association in Europe (185.000 members) has used Vocdoni to organize its Annual General Meetings and elections 2 years in a row and will handle a new election on January 2021.

- Do you require feedback or data from attendees post-event?

> Feedback will not only be welcome, but will be very important. Especially to continue improving the tool to cover the governance needs of Devcon and other community events. This is critical, as governance implies dealing with human behaviour and this first pilot will certainly priovide many inputs to improve the product. We'll have available a helpdesk (chat+chatbot+KnowledgeBase+ticketsystem) and we could establish extra channels (e.g. telegram) to receive technical, accessability and UX/UI feedback.

Reviewer: ```suggestion
Feedback will not only be welcome, but will be very important. Especially to continue improving the tool to cover the governance needs of Devcon and other community events. This is critical, as governance implies dealing with human behaviour and this first pilot will certainly priovide many inputs to improve the product. We'll have available a helpdesk (chat+chatbot+KnowledgeBase+ticketsystem) and we could establish extra channels (e.g. telegram) to receive technical, accessebility and UX/UI feedback.
```
